src/bci_build/package.py

```diff
@@ -770,6 +770,11 @@ def pkg_filter_func(p: Union[str, Package]) -> bool:
             for pkg_type in PKG_TYPES
         )
 
+        # Ensure that we build base OS containers with the proper product flavor
+        # for correct telemetry reporting
+        if self.is_opensuse and isinstance(self, OsContainer):
+            bootstrap_packages.append("openSUSE-release-appliance-docker")
+
```

Comment:
No, please no additional hidden magic. I fear that this will be impossible to find down the road why packages are suddenly getting added in some containers and not in others.

Reply:
well, in this case it is easy to grep (just search for the package name)